torch_xla/csrc/tensor.cpp

```diff
@@ -367,6 +367,27 @@ class XLATensor::DeviceContextArena {
     devctx->seed_ir_value = torch::lazy::Value();
   }
 
+  torch::lazy::BackendDataPtr GetRngSeedData(
+      const torch::lazy::BackendDevice& device, bool reset) {
```

Comment:
Can you explain a bit how this 'reset' arg works? In dynamo if we want to replace the seed placeholder with the real seed, should we call this method with reset to be true?

Reply:
if you pass `reset == false`, and keep calling `GetRngSeedData` it will always return you the same seed data. If you pass `reset==true`, it will give you a new seed.

However `mark_step` will also increment the base seed, and I believe we call `mark_step` for every `optim_mode` now. If that's the case we don't need to `reset` the seed in this API.